frontend/src/components/AddRecipeForm.js

```diff
@@ -151,12 +173,57 @@ const AddRecipeForm = () => {
                 </div>
               ))}
             </div>
+            <div className="row d-flex align-items-center">
+              <p className="col-2 mt-2 pl-0 d-flex align-items-center">
+                Kategória hozzáadása
+              </p>
+              <div className="col-6">
+                <input
+                  className="form-control"
+                  id="newCategory"
+                  value={newCategory}
+                  type="text"
+                  onChange={(e) => setNewCategory(e.target.value)}
+                />
+              </div>
+              <button
+                className="btn btn-success"
+                onClick={() => addCategory(newCategory)}
+                data-toggle="modal"
+                data-target="#myModal"
+              >
+                +
+              </button>
+            </div>
           </div>
         </div>
-
         <button className="btn btn-success" type="submit">
           Hozzáadás
         </button>
+        <div class="modal fade" id="myModal">
+          <div class="modal-dialog">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h4 class="modal-title">Konyhatündér üzenet</h4>
+                <button type="button" class="close" data-dismiss="modal">
+                  &times;
+                </button>
+              </div>
+
+              <div class="modal-body">{error}</div>
+
+              <div class="modal-footer">
+                <button
+                  type="button"
+                  class="btn btn-danger"
+                  data-dismiss="modal"
+                >
+                  Close
```

Comment:
Ez szerintem legyen magyar, ha már a felület többi része is az.

Reply:
Done.